triplea_maps.yaml

```diff
@@ -1924,7 +1924,7 @@
     <br>
     House of Habsburg Beta Version
     <br>Large map of Europe set in the 1500s
-- mapName: CrazyG's Big World War One
+- mapName: crazygs_ww1
```

Comment:
Errm are you sure about that?
This is the display name in the engine.

Reply:
No I'm not sure at all.

The current version is downloading wrong. The master file for the zip has a wrong name, I'm clueless where it comes from and I couldn't find any help in the forum.

Reply:
its downloading with the outermost folder as crazyg's_big_world_war_one.

I'm rather confused by how many different names come up in the process of uploading these maps

Reply:
It's downloading correctly for me. Are you sure you're looking at the correct file?
If you are try deleting it and try again.

Reply:
I'll try again, give me one second

Reply:
I tried downloading the game again, it won't launch.

The folder structure is right, however the outermost folder, the zipped master folder, has a name I don't want it to have. When you select the game to play (you can view the game notes normally), you get an error. I'm under the impression its because this folder is named incorrectly, and it works after changing the name. I don't know where the name for this folder is set, I've followed the map upload process to every point with a name, editing this YAML was my last idea.

Reply:
Ahh I got what's the problem now.
Your Idea was indeed correct. However it's not using the exact name specified here, but instead taking this name, converting it all to lowercase, and replacing spaces with underscores.
So we can have a second iteration of this, where you capitalize the name so it looks nicer.

Reply:
```suggestion
- mapName: CrazyGs WW1
```